VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -208,12 +211,21 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
                     if let pipController = self.pictureInPictureController as? AVPictureInPictureController {
                         return pipController
                     } else {
-                        guard
-                            let layer = videoView?.playerLayer,
-                            props.pictureInPictureActive else { return nil }
+                        guard let layer = videoView?.playerLayer else { return nil }
                         let pipController = AVPictureInPictureController(playerLayer: layer)
                         pipController?.delegate = self
                         self.pictureInPictureController = pipController
+                        if let pictureInPictureController = self.pictureInPictureController {
```

Comment:
`pipController` is optional, that's why we have verification here. If you worrying about bad code readability, I propose to replace with: `if let pipController = pipController {`